### ISSUE TITLE
please update RM training readme

### ISSUE THREAD

**Issue by akk-123** [LAION-AI/Open-Assistant#2136]
reward model training is update by #2071 , please update readme of RM model

**Comment by CloseChoice** [comment]
You are right that this needs an update. One current problem is, that we just have a private dataset for RM. But more are coming soon, see https://github.com/LAION-AI/Open-Assistant/issues/2174
